PWGHF/Tasks/taskMcEfficiencyToXiPi.cxx

```diff
@@ -34,6 +37,9 @@ using namespace o2::framework;
 using namespace o2::framework::expressions;
 
 struct HfTaskMcEfficiencyToXiPi {
+
+  o2::hf_evsel::HfEventSelectionMc hfEvSelMc; // mc event selection and monitoring
```

Comment:
I'm not sure if this is enough, you probably need to have also an
```
o2::hf_evsel::HfEventSelection hfEvSel;
```
to inject the configurables in the task